.github/workflows/conformance.yml

```diff
@@ -0,0 +1,86 @@
+name: Manually run upstream conformance test on Linux
+run-name: Run ${{ inputs.test-suite }} test with Antrea ${{ inputs.antrea-version }} and K8s ${{ inputs.k8s-version }}
```

Comment:
will this give a weird name if `inputs.antrea-version` or `inputs.k8s-version` is empty?

Reply:
actually not that weird based on https://github.com/tnqn/antrea/actions/runs/5129098655, looks like the extra whitespaces are automatically removed